rosidl_runtime_c/src/type_description/field__description.c

```diff
@@ -98,9 +98,11 @@ rosidl_runtime_c__type_description__Field__get_type_description(
     {rosidl_runtime_c__type_description__Field__REFERENCED_TYPE_DESCRIPTIONS, 1, 1},
   };
   if (!constructed) {
+    #ifndef NDEBUG
     assert(0 == memcmp(&rosidl_runtime_c__type_description__FieldType__EXPECTED_HASH, rosidl_runtime_c__type_description__FieldType__get_type_hash(NULL), sizeof(rosidl_type_hash_t)));
     description.referenced_type_descriptions.data[0].fields = rosidl_runtime_c__type_description__FieldType__get_type_description(NULL)->type_description.fields;
     constructed = true;
+    #endif
```

Comment:
```suggestion
    #ifndef NDEBUG
    assert(0 == memcmp(&rosidl_runtime_c__type_description__FieldType__EXPECTED_HASH, rosidl_runtime_c__type_description__FieldType__get_type_hash(NULL), sizeof(rosidl_type_hash_t)));
    #endif
    description.referenced_type_descriptions.data[0].fields = rosidl_runtime_c__type_description__FieldType__get_type_description(NULL)->type_description.fields;
    constructed = true;
```

The same goes below.

Reply:
My bad, fixed it here: https://github.com/ros2/rosidl/pull/739/commits/8a70ee16fd5d2a8e6d80a1264fba0b86bf6ac434.

Not sure why the CI is failing thought.

Reply:
LGTM. Let's wait for others' opinions.